config/linux-nitropad-x.config

```diff
@@ -0,0 +1,336 @@
+CONFIG_LOCALVERSION="-heads"
```

Comment:
Please use helpers to save in oldefconfig and compare with other boards to tune accordingly if needed

https://github.com/osresearch/heads/blob/3747d58510c7dde30a75cef4aba7977c4cbbfa7b/modules/linux#L233

Reply:
done